public/docs/ts/latest/guide/displaying-data.jade

```diff
@@ -179,56 +180,56 @@ figure.image-display
+  * declares a public property of the same name
+  * initializes that property with the corresponding argument when we "new" an instance of the class
 
 .l-main-section
 :marked
-  ## Use the Hero class
-  Let's redefine the heroes property in our component to return an array of these Heroes
+  ## Using the Hero class
+  Let's redefine the `heroes` property in our component to return an array of these Hero objects
   and also set the `myHero` property with the first of these mock heroes.
 +makeExample('displaying-data/ts/app/app.component.3.ts', 'heroes', 'app.component.ts (excerpt)')(format=".")
 
 :marked
   We'll have to update the template.
-  At the moment it displays the entire hero object which used to be a string value.
-  Let's fix that so we interpolate the `hero.name` property
+  At the moment it displays the entire `hero` object, which used to be a string value.
+  Let's fix that so we interpolate the `hero.name` property.
 +makeExample('displaying-data/ts/app/app.component.3.ts', 'template','app.component.ts (template)')(format=".")
 
 :marked
-  Our display looks the same but we know how much better it is under the hood.
+  Our display looks the same, but now we know much better what a hero really is.
 
 <a id="ngIf"></a>
 .l-main-section
 :marked
   ## Conditional display with NgIf
 
-  Sometimes the app should display a view or a portion of a view only under prescribed circumstances.
+  Sometimes the app should display a view or a portion of a view only under specific circumstances.
 
-  In our example, we'd like to display a message if we have a large number of heroes ... say more than 3.
+  In our example, we'd like to display a message if we have a large number of heroes—say, more than 3.
 
-  The Angular `NgIf` directive will insert or remove an element based on a truthy/falsey condition.
+  The Angular `NgIf` directive inserts or removes an element based on a truthy/falsey condition.
   We can see it in action by adding the following paragraph at the bottom of the template:
 +makeExample('displaying-data/ts/app/app.component.ts', 'message')
 .alert.is-important
   :marked
-    Don't forget the leading asterisk (\*) in front of `*ngIf`. It is an essential part of the syntax.
+    Don't forget the leading asterisk (\*) in `*ngIf`. It is an essential part of the syntax.
     Learn more about this and `NgIf` in the [Template Syntax](./template-syntax.html#ngIf) chapter.
 
 :marked
   The [template expression](./template-syntax.html#template-expressions) inside the double quotes
-  looks much like JavaScript and it is much like JavaScript.
+  looks much like JavaScript and it _is_ much like JavaScript.
   When the component's list of heroes has more than 3 items, Angular adds the paragraph to the DOM and the message appears.
-  If there were 3 or fewer items, Angular omits the paragraph and there is no message.
+  If there are 3 or fewer items, Angular omits the paragraph, so no message appears.
```

Comment:
The mix of past/conditional and present tense (were...omits...is) was what I was mainly trying to fix. The relationship of the message to the paragraph/Angular was also a bit unclear. (The next sentence does address that, iirc, but better to be clearer from the start.) You can probably think of a better solution, though.